src/qt/veil/forms/settingsmintingauto.ui

```diff
@@ -218,7 +218,7 @@ font-size:16px;</string>
 font-size:16px;</string>
         </property>
         <property name="text">
-         <string>100000</string>
+         <string>100000</string> // What is this about?
```

Comment:
Is this comment syntax correct...? Reference: [The .ui File Format](https://doc.qt.io/archives/3.3/designer-manual-16.html#1-4)

Reply:
Can't comment off line that way? Removed.